src/pages/app-development/starter-kit/actions-overview.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Actions overview
+description: Learn about the actions available in Starter Kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Actions overview
+
+The Adobe Commerce Extensibility Starter Kit includes boilerplate code and samples for the following types of actions:
+
+* `consumer`
+* `event handler`
+* `event ingestion`
+* `synchronous webhook`
+
+The `consumer` and `event handler` actions implement the business logic needed to synchronize data between Commerce and the external systems being integrated.
+
+Additionally, boilerplate code, responses, and samples for event ingestion and synchronous webhook actions are provided.
+
+<InlineAlert variant="info" slots="text"/>
+
+The responses for each action are configured for the [Commerce webhook module](https://developer.adobe.com/commerce/extensibility/webhooks/). To customize the responses for other implementations, modify the `response.js` file.
+
+## `consumer` action
+
+The main purpose of this action is to route events to the [`event handler` action](#event-handler-action). Normally, this routing is determined by the name of the event received.
+
+The `consumer` action is subscribed to a set of events. In many cases, all the events originate from the same entity, such as the `product` entity. However, there are examples where a consumer receives events from multiple entities belonging to the same "domain", such as `order` and `shipment`. When the event provider receives an event, this runtime action will be automatically activated.
+
+The response returned by a `consumer` action is expected to be consistent with the response received from the activation of the subsequent `event handler` action. For example, if the `event handler` action returns an `HTTP/400` status, the consumer action is expected to respond with the same status.
```

Comment:
If you want to add additional details about this point. That consistency is built in to ensure proper event processing retry based on the response received from the `event handler` action.